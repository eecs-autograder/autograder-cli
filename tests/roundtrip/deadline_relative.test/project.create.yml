project:
  name: Test Project
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    deadline:
      cutoff_type: relative
      deadline: Jan 8, 2024 7:59pm
      cutoff: 1d 4h 2m
