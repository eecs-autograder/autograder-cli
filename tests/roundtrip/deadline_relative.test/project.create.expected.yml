project:
  name: Test Project
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    deadline:
      cutoff_type: relative
      deadline: Jan 08, 2024 07:59PM
      cutoff: 1d4h2m
